uwrt_mars_rover_utils/uwrt_mars_rover_estop/src/estop.cpp

```diff
@@ -0,0 +1,76 @@
+#include <uwrt_mars_rover_estop/estop.hpp>
+
+namespace composition{
```

Comment:
namespace should not be `composition`, something more relevant, such as `uwrt_mars_rover_estop`